filters/include/pcl/filters/experimental/passthrough.h

```diff
@@ -0,0 +1,215 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2021-, Open Perception
+ *
+ *  All rights reserved
+ */
+
+#pragma once
+
+#include <pcl/filters/experimental/functor_filter.h>
+
+#include <limits> // for numeric_limits
+
+namespace pcl {
+namespace experimental {
+
+template <typename PointT>
+struct PassThroughFunctor {
+  PassThroughFunctor() = default;
+  PassThroughFunctor(const uindex_t* field_offset,
+                     const float* filter_limit_min,
+                     const float* filter_limit_max,
+                     const bool* has_filter_field,
+                     const bool* negative)
+  : field_offset_(field_offset)
+  , filter_limit_min_(filter_limit_min)
+  , filter_limit_max_(filter_limit_max)
+  , has_filter_field_(has_filter_field)
+  , negative_(negative)
+  {}
+
+  bool
+  operator()(const PointCloud<PointT>& cloud, index_t idx) const
```

Comment:
Can we add the entire reasoning behind return negative here as a comment on the actual behavior of this function object?

Reply:
Done, I added a few comments alongside the code